lib/services/android-project-service.ts

```diff
@@ -331,6 +334,25 @@ class AndroidProjectService implements IPlatformProjectService {
 		}
 	}
 
+	private getAndroidTarget(frameworkDir: string): IFuture<string> {
+		return ((): string => { 
+			let newTarget = this.$options.sdk ? `${AndroidProjectService.ANDROID_TARGET_PREFIX}-${this.$options.sdk}` : this.getLatestValidAndroidTarget(frameworkDir).wait();
+			if(!_.contains(this.SUPPORTED_TARGETS, newTarget)) {
+				let versionNumber = parseInt(_.last(newTarget.split("-")));
+				if(versionNumber) {
+					let minSupportedVersion = _.first(this.SUPPORTED_TARGETS.sort());
+					this.$logger.trace(`Minimum supported Android target is ${minSupportedVersion}.`);
+					if(versionNumber < parseInt(_.last(minSupportedVersion.split("-")))) {
```

Comment:
If we decide to support at least 19, we should change the constant and everything will work as expected (on theory of course :smile_cat: ) but the code will be more readable and will avoid unneeded operations :smile: